lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -576,9 +577,6 @@ public static function get_user_data() {
 		$theme_json = apply_filters( 'wp_theme_json_data_user', new WP_Theme_JSON_Data_Gutenberg( $config, 'custom' ) );
 		$config     = $theme_json->get_data();
 
-		// Needs to be set for schema migrations of user data.
```

Comment:
Additionally, we can add back the changes at https://github.com/WordPress/gutenberg/pull/61262:

```php
$theme_json = apply_filters( 'wp_theme_json_data_user', new WP_Theme_JSON_Data_Gutenberg( $config, 'custom' ) );
static::$user = $theme_json->get_theme_json();

return static::$user;
```

Reply:
done it myself